core/java/iesi-core/src/test/java/io/metadew/iesi/common/crypto/JavaKeystoreTest.java

```diff
@@ -58,7 +58,7 @@ public void testEncryptWrongAlias() throws Exception {
             factory.getKeySpec(
                     ske.getSecretKey(), PBEKeySpec.class);
         } catch (NullPointerException e) {
-            assertEquals(null, e.getMessage());
+            assertThat(e.getMessage()).isNull();
         }
     }
```

Comment:
There exists an AssertJ test method that allows for testing if a certain exception is thrown